opds.py

```diff
@@ -62,6 +62,10 @@ def default_lane_url(cls):
     def top_level_title(self):
         return "All Books"
 
+    def facet_url(self, facets):
```

Comment:
I guess I'm not sure what the urls will look like for the static feeds, but if they have query parameters we'll need something to map those to the appropriate static feed. I was thinking the static feeds would do something else.